src/relentless/simulate/hoomd.py

```diff
@@ -1228,11 +1240,27 @@ def act(self, timestep):
                         for i, j in self._rdf:
                             query_args = dict(self._rdf[i, j].default_query_args)
                             query_args.update(exclude_ii=(i == j))
+                            neighbors = (
+                                aabbs[j]
+                                .query(
+                                    snap.particles.position[type_masks[i]], query_args
+                                )
+                                .toNeighborList()
+                            )
+                            if snap.bonds is not None:
```

Comment:
Here you would add a check if bonds are in the exclusion list as well. Probably you need to have that forwarded to this action though, since we would normally grab it from the `sim`.